lib/vimeo.js

```diff
@@ -158,10 +185,11 @@ Vimeo.prototype._handleRequest = function (callback) {
     })
 
     if (res.statusCode >= 400) {
-      // Failed api calls should wait for the response to end and then call the callback with an
+      // Failed api calls should wait for the response to end and then call the callback or the reject fn if passed in with an
       // error.
       res.on('end', function () {
         const err = new Error(buffer)
+        callback = reject || callback
```

Comment:
it's more clear to redefine `reject` earlier / first line of the method as `reject = reject || callback` maybe?

Reply:
makes sense, updated!